ci/vale/styles/config/vocabularies/nat/accept.txt

```diff
@@ -115,6 +118,7 @@ NIM(s?)
 npm
 NumPy
 NAT's
+NASSE's
```

Comment:
In general we don't allow for inanimate objects to possess things. (yes I know `NAT's` shouldn't be in this list either)